src/Microsoft.Health.Fhir.Api/Features/Filters/ValidateExportRequestFilterAttribute.cs

```diff
@@ -46,12 +53,14 @@ public override void OnActionExecuting(ActionExecutingContext context)
                 throw new RequestNotValidException(string.Format(Resources.UnsupportedHeaderValue, PreferHeaderName));
             }
 
-            // Validate that the request does not contain any query parameters.
+            // Validate that the request does not contain query parameters that are not supported.
             IQueryCollection queryCollection = context.HttpContext.Request.Query;
-            if (queryCollection?.Count > 0)
+            foreach (string paramName in queryCollection?.Keys)
             {
-                string paramName = queryCollection.FirstOrDefault().Key;
-                throw new RequestNotValidException(string.Format(Resources.UnsupportedParameter, paramName));
+                if (!_supportedQueryParams.Contains(paramName))
```

Comment:
Does this account for casing? Should _SINCE be valid or invalid?

Reply:
Yes, _SINCE will be invalid. I am initializing the HashSet with a `StringComparer.Ordinal` comparison so that will take care of the proper comparison. I will also add a UT that covers that case.

Reply:
Added some more UTs to cover this scenario